File: app/src/main/java/sabalan/paydar/mohtasham/ezibazi/model/State.java
package sabalan.paydar.mohtasham.ezibazi.model;

import org.json.JSONException;
import org.json.JSONObject;

public class State {
  private int id;
  private String name;

  public static class Parser{

    public static State parse(JSONObject object){
      State state = new State();
      try {
        state.setId(object.getInt("id"));
        state.setName(object.getString("name"));
      } catch (JSONException e) {
        e.printStackTrace();
      }
      return state;
    }
  }



  public int getId() {
    return id;
  }

  public void setId(int id) {
    this.id = id;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }
}

File: app/src/main/java/ir/easybazi/model/State.kt
package ir.easybazi.model

import org.json.JSONException
import org.json.JSONObject

class State {
    var id: Int = 0
    var name: String? = null

    object Parser {

        fun parse(`object`: JSONObject): State {
            val state = State()
            try {
                state.id = `object`.getInt("id")
                state.name = `object`.getString("name")
            } catch (e: JSONException) {
                e.printStackTrace()
            }

            return state
        }
    }
}
